geometry_msgs/Point initial_point
geometry_msgs/Point landing_point
caramel_kinematics/AllLegPoints leg_points
float64 period
float64 height
int32 resolution

float64 resolution_first_fraction 1.0
float64 period_first_fraction 1.0
---
geometry_msgs/Point[] points
builtin_interfaces/Duration[] time_from_start